lib/datasource/remote/model/token_model.dart

```diff
@@ -149,7 +150,7 @@ class TokenModel extends Equatable {
 }
 
 const seedsToken = TokenModel(
-  chainName: "Telos",
+  chainName: SeedsChains.telos.value,
```

Comment:
Seems we can't use extensions in constant expressions. Ref explanation here: https://github.com/dart-lang/language/issues/663

Reply:
Maybe you need to update your flutter version?

Reply:
Oh you're right - there's something weird going on with this PR.